test/cases/generation.sh

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/bash
+
+#
+# The objective of this script is to provide special requirement needed
+# for generate-test-cases. For example, disable the dnf-json daemon
+# and then execute generate-test-cases.
```

Comment:
Not exactly sure what this test does but looking at the history I found:
```
Test that the test cases generation will properly start the
dnf-json.socket when needed.
```
So I would guess it's doing just that, not sure if more uses are expected.

Reply:
I asked @lavocatt, and this is what I understood. If it's not clear enough we can think of another way to rephrase it. I think `generate-test-cases` needs dnf-json to be disabled on start so it can enable it. This script just disables this service